portal-api/Dockerfile

```diff
@@ -2,39 +2,17 @@ FROM node:lts AS image
 
 WORKDIR /app
 
-ADD package.json .
+ADD . .
```

Comment:
This doesn't look right to me.

Reply:
how so? This just adds all the files from the package into the dockerfile. Previously only the package file was being copied, then other files selectively. Is there something better we should be doing?

Reply:
I think this should be ok as node_modules and dist folders that the docker image would generate are in .gitignore so the local ones won't conflict with the docker ones. There will be some files in docker image that docker doesn't need but I don't think that matters for local dev environment.